azure/deploy-to-azure/azure-integration/azuredeploy-managementgroups.json

```diff
@@ -100,6 +100,13 @@
                 "description": "Whether or not to use custom metrics"
             }
         },
+        "excluded_resource_providers": {
+            "type": "array",
```

Comment:
its a string, right?

Reply:
so excluded resource providers is a list of string within the json. So kept it as an array here. I understand json is also represented by a string. Dont have a strong preference either way.